src/main/blackbox/blackbox.c

```diff
@@ -641,7 +642,17 @@ static void writeIntraframe(void)
     blackboxWriteUnsignedVB(blackboxCurrent->time);
 
     if (testBlackboxCondition(CONDITION(COMMAND))) {
-        blackboxWriteSigned16VBArray(blackboxCurrent->command, 4);
+        // Write roll, pitch and yaw first:
+        blackboxWriteSigned16VBArray(blackboxCurrent->command, 3);
```

Comment:
This is incorrect. Above it is defined that there are 4 fields of type TAG8_4S16. But here we are writing only 3.